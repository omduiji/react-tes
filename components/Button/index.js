import React from 'react';
import styles from './styles.module.scss';

const Button = ({ children, handleClick }) => (
  <button type="button" onClick={handleClick}>
    {children}
  </button>
);

export default Button;
