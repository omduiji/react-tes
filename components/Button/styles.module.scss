.button {
    outline: none;
    background-color: white;
    color: #333;
    border: 1px solid #aaa;
    height: 40px;
    display: flex;
    justify-content: center;
    align-items: center;
    padding: 0 20px;
    border-radius: 4px;
    margin: 0.75em 0;

    &:hover {
        background-color: #f8f8f8;
    }
}
