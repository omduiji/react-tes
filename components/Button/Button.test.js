import React from 'react'
import { render } from '@testing-library/react'
import Button from '.'

test('should render Button', () => {
  const { asFragment } = render(<Button />)

  expect(asFragment()).toMatchSnapshot()
})
