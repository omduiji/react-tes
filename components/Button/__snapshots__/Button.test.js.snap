// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Button 1`] = `
<DocumentFragment>
  <button
    type="button"
  />
</DocumentFragment>
`;
